src/padplus-manager/api-request/media-data-request.ts

```diff
@@ -0,0 +1,40 @@
+interface FuncObj {
```

Comment:
This file should be named as `queue` or something similar. Because this file contains only one class named `Queue`.